acceptance/ui/features/api/cli.rb

```diff
@@ -0,0 +1,327 @@
+require "json"
+require "capybara/rspec"
+
+#
+# Encapsulates the CLI functionality to be exposed to the
+# acceptance tests: CC.CLI.<function>
+#
+class CLI
```

Comment:
Having all of the generic CLI processing consolidated in one class is a good thing, but I worry about this class becoming unmanageably large as we add more and more methods to interface with more and more commands. Would it make more sense to have a different subclasses for different commands. Something like HostCLI for the host commands, PoolCLI for the pool commands etc?